news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,127 @@
+
+Overall, the map is quite challenging for its star rating, being harder than other mapsets people usually see in the Ranked category. It also features several mappers, providing players with different styles across all difficulties. It's quite the engaging map, and one that could surely make you wiggle from joy!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shelter.jpg)](https://osu.ppy.sh/s/536749)
+
+Despite this set is being **[Myle](https://osu.ppy.sh/users/4915649)**'s very first ranked set it came out to be a really enjoyable one. The spread built up to grew quite constantly. He learned how to make the rhythm to be consistent and yet still provide variable patterns in a short time. The flow of the difficulties are just splendid and perfectly executed. The Platter is really made for the targeted audience, he held back with the HyperJumps and only placed one where it should be which gives better impact in overall and feels really good. It was really enjoying to play, he made a good work!
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/trigger.jpg)](https://osu.ppy.sh/s/555076)
+
+This beatmap set by **[Wonki](https://osu.ppy.sh/users/7898495)** definitely is a fun experience from the ranked pool and congratulations to Wonki for winning this month spotlights and also for providing us such a good piece. The scroll speed changes on this map feels intense but they are not overdone and it just feels natural with the song, making it really confortable to play. Combine that with a cool usage of minijacks at the very end with some simple long note patterns this map just has everything you want to be the map of the month... and its art!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/re-lm.jpg)](https://osu.ppy.sh/s/588066)
+
+This map has a heavy use of scroll speed changes, which are thoughtfully crafted to be intuitive and smooth. It is quite impressive given how easy to cross the line of being more of a distraction than a feature. Although the scroll speed change is particularly not the reason why this map is deserve to be in the Spotlights. **[Lude](https://osu.ppy.sh/users/6974536)** doesn't shy away from long note usage in their mapping of this track. Instead, he use it to add atmospheric value, complexity and originality, which is great to see. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves, makes this a suitable pick for May's spotlight.
```

Comment:
Parachor requested a change on this to:

This map is eye-catching for a few reasons: The most obvious being the heavy use of scroll speed variations, which are thoughtfully crafted to be intuitive and smooth. This is quite impressive given how easy it is to cross the line of being more of a distraction than a feature. However, the scroll speed variations are not the main reason this map is particularly deserving of a spotlight. **[Lude](https://osu.ppy.sh/users/6974536)** doesn't shy away from long note usage in their mapping of this track. Instead, they use it to add atmospheric value, complexity, and originality, which is great to see in the ranked section. These two points combined with some enjoyable jack-oriented mapping and very smooth difficulty curves makes this a suitable pick for May's spotlight.